cheatsheets/gleam-for-elixir-users.md

```diff
@@ -94,8 +94,9 @@ let size = 1
 #### Elixir
 
 ```elixir
-[x] = [1] # assert `x` is a 1 element list and assign it to this element's value
-2 = x # runtime error because x's value is 1
+[x, y] = [1, 2]
+2 = y # assert that y is 2
+2 = x # runtime error
```

Comment:
```suggestion
[x, y] = [1, 2] # assign 1 to x and 2 to y by pattern matching on the list [1, 2].
2 = y # assert that y is 2
2 = x # runtime error because x's value is 1
```

Here's a suggestion for adding comments as you asked. I'm fairly sure that the old comment:

```
[x] = [1] # assert `x` is a 1 element list and assign it to this element's value
```

is just wrong (`[x] = [1]` doesn't make any assertions about the current value of x, and doesn't cause x to be a list), so I don't think it should be restored.

Reply:
Ah yes, it should "assert the value is a 1 element list"

Reply:
How about this?

```suggestion
[x, y] = [1, 2] # assert that the value is a list of length 2 and assign the first element to x and the second to y
2 = y # assert that y is 2
2 = x # runtime error because x's value is 1
```

Reply:
Sounds good!